src/track-promise.cc

```diff
@@ -0,0 +1,72 @@
+#include "env.h"
+#include "env-inl.h"
```

Comment:
@trevnorris Actually, it seemed to function without anything being included. I really don't know why?